apps/scan/frontend/src/screens/poll_worker_screen.test.tsx

```diff
@@ -233,7 +232,6 @@ describe('transitions from polls paused', () => {
     apiMock.expectSetPollsState('polls_open');
     apiMock.expectGetConfig({ pollsState: 'polls_open' });
     userEvent.click(screen.getByText('Yes, Resume Voting'));
-    await screen.findByText('Resuming Voting…');
```

Comment:
This is among a few cases where loading states are no longer visible in tests without setting up some sort of `deferred` promise, and it just didn't seem worth the trouble.

Reply:
Do we have other coverage of this? I think one of the main responsibilities of the frontend tests should be covering the various states we expect to see.

I get that it feels tedious when it takes a lot of work. Maybe that just means we need to make it easier to set up

Reply:
Actually I was wrong, I think I was just avoiding an `act` warning here. Added back the assertions, wrapped. a0b04d0b9